app/assets/stylesheets/patterns/_buttons.scss

```diff
@@ -1,29 +1,16 @@
 .button, input[type="submit"] {
-  background: $dark-grey;
-  border: none;
+  background: $pale-prune;
   border-radius: 2px;
   color: $offwhite;
-  display: block;
   font-size: 15px;
-  letter-spacing: 0.009em;
-  margin-left: 2%;
-  padding: 8px 16px;
+  padding: 16px;
   text-transform: uppercase;
```

Comment:
Can this one go to the bottom to keep alpha order?

Reply:
👍